pelican_katex/plugin.py

```diff
@@ -37,6 +37,9 @@ def configure_pelican(plc):
     if "KATEX_PREAMBLE" in plc.settings:
         rendering.KATEX_PREAMBLE = plc.settings["KATEX_PREAMBLE"]
 
+    if "KATEX_MATHML" in plc.settings:
```

Comment:
`KATEX_MATHML` might not be the best name for this. KaTeX can also output MathML after all.